tests/test_Config.cpp

```diff
@@ -269,36 +269,40 @@ BOOST_AUTO_TEST_CASE(test_TempFile) {
     BOOST_CHECK(!rwfs::exists(path));
 }
 
-BOOST_AUTO_TEST_CASE(test_config_initial) {
+BOOST_AUTO_TEST_CASE(test_configparser_initial) {
```

Comment:
Maybe extra `_` after `test_config`?

Reply:
No, I test the configparser here (=configuration file parser). There are also tests for the argparser (=argument parser) in there.

Reply:
So camel case?

Reply:
camel case will look good here.